src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -153,13 +153,15 @@ public function init_form_fields()
     public function getIconUrl ()
     {
         $svg = $this->iconFactory()->svgUrlForPaymentMethod($this->getMollieMethodId());
+
         return $svg;
     }
 
 	/**
 	 * @return string
 	 */
-	public function getIssuerIconUrl( $issuer_id ) {
+	public function getIssuerIconUrl( $issuer_id )
+    {
```

Comment:
Is this additional space before the curly brace because of tab and not space or ???